File: recaudacion/2.Codigo/frontend/src/app/core/models/tarifario.ts
export class Tarifario {
    tarifarioId: number;
    clasificadorIngresoId: number;
    codigo: string;
    nombre: string;
    porcentajeUit: number;
    precio: number;
    grupoRecaudacionId: number;
    grupoRecaudacion: any;
    precioVariable: boolean;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    codigoNombre: string;
    index: number;
}

export class TarifarioFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn: string;
    sortOrder: string;
    clasificadorIngresoId: number;
    codigo: string;
    nombre: string;
    estado: boolean;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiComprobantePago/Domain/Tarifario.cs
namespace RecaudacionApiComprobantePago.Domain
{
    public class Tarifario
    {
        public int TarifarioId { get; set; }
        public int ClasificadorIngresoId { get; set; }
        public string Codigo { get; set; }
        public string Nombre { get; set; }

    }
}